src/js/components/ServicesTable.js

```diff
@@ -294,6 +295,11 @@ var ServicesTable = React.createClass({
     let serviceStatus = service.getStatus();
     let serviceStatusClassSet = StatusMapping[serviceStatus] || '';
     let taskSummary = service.getTasksSummary();
+    let {TASK_RUNNING = 0} =
+        MesosSummaryStore.getFrameworkTasksByName(service.getName());
```

Comment:
Let's not use the `CompositeState`, as we've decided to deprecate it with the introduction of the `DCOSStore`.  How about merging  the data within the `DCOSStrore`?